source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -987,240 +987,247 @@ TopologyType ReplicaSetWithPrimary
     run `updateRSWithPrimaryFromMember`_.
 
 Actions
-~~~~~~~
-
-.. _updateUnknownWithStandalone:
+```````
 
 updateUnknownWithStandalone
-  This subroutine is executed
-  with the ServerDescription from Standalone
-  when the TopologyType is Unknown::
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subroutine is executed with the ServerDescription from Standalone
+when the TopologyType is Unknown
```

Comment:
Here and for the other code blocks let's keep a single `:`, like:
```
... when the TopologyType is Unknown:

.. code-block:: python
```

Reply:
Agreed, fixed!